service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -71,18 +80,32 @@ public static function fromArray(mixed $data): self
      *     lastName: string,
      *     dob: string,
      *     address: string[],
-     *     lpas: string[]
+     *     lpas: string[],
+     *     kbvQuestions?: string,
+     *     documentComplete: bool
      * }
      */
     public function toArray(): array
     {
-        return [
+        $arr = [
             'personType' => $this->personType,
             'firstName' => $this->firstName,
             'lastName' => $this->lastName,
             'dob' => $this->dob,
             'address' => $this->address,
             'lpas' => $this->lpas,
+            'documentComplete' => $this->documentComplete,
         ];
+
+        if ($this->kbvQuestions !== null) {
+            $arr['kbvQuestions'] = $this->kbvQuestions;
```

Comment:
Think kvbQuestions was initially missing as we had not started on the questions part yet, also because I could not think of what validation we would do here, did not add this subsequently but it makes sense to be here for completeness if we go ahead to pass CaseData as a whole object to the views